micrometer-core/src/main/java/io/micrometer/core/instrument/binder/netty4/NettyEventExecutorMetrics.java

```diff
@@ -77,18 +78,26 @@ public NettyEventExecutorMetrics(Iterable<EventExecutor> eventExecutors, Iterabl
 
     @Override
     public void bindTo(MeterRegistry registry) {
+        if (this.eventExecutors instanceof MultithreadEventLoopGroup) {
+            MultithreadEventLoopGroup eventLoopGroup = (MultithreadEventLoopGroup)this.eventExecutors;
```

Comment:
```suggestion
            MultithreadEventLoopGroup eventLoopGroup = (MultithreadEventLoopGroup) this.eventExecutors;
```
lint? 로컬에서 린트 체크 한번 돌려주시면 좋을거같습니다!